src/css/styles.css

```diff
@@ -181,4 +180,22 @@ h4 {
 p {
   font-weight: 200;
   font-size: 1em;
-}
\ No newline at end of file
+}
+
+/*Media Queries: Small screens and tablets*/ 
+
+@media only screen and (max-width: 900px) {
+  .grid {
+    column-count: 2;
+    column-gap: 1em;
+    column-fill: balance;
+  }
+}
+
+@media only screen and (max-width: 650px) {
+  .grid {
+    column-count: 1;
+    column-gap: 1em;
+    column-fill: balance;
+  }
+}
```

Comment:
Nice work! This looks awesome and I think that the shifting from 3 to 2 to 1 columns is definitely the best way to go about this!